checkov/common/util/type_forcers.py

```diff
@@ -115,3 +115,23 @@ def convert_csv_string_arg_to_list(csv_string_arg: list[str] | str | None) -> li
         return csv_string_arg[0].split(',')
     else:
         return csv_string_arg
+
+
+def convert_prisma_policy_filter_to_dict(filter_string: str) -> Dict[Any, Any]:
+    """
+    Converts the filter string to a dict. For example:
+    'policy.label=label,cloud.type=aws' becomes -->
+    {'policy.label': 'label1', 'cloud.type': 'aws'}
+    Note that the API does not accept lists https://prisma.pan.dev/api/cloud/cspm/policy#operation/get-policies-v2
+    This is not allowed: policy.label=label1,label2
+    """
+    filter_params = {}
+    if filter_string:
+        try:
+            for f in filter_string.split(','):
+                f_name = f.split('=')[0]
+                f_value = f.split('=')[1]
```

Comment:
```suggestion
                f_name, f_value = f.split('=')
```
if you are sure, that there will be only one `=`, otherwise
```suggestion
                f_name, f_value, *_ = f.split('=')
```
then it will throw away excessive parts

Reply:
Accepted option 1 since no more `=` will be present